addons/map_gestures/functions/fnc_initDisplayDiary.sqf

```diff
@@ -0,0 +1,75 @@
+#include "script_component.hpp"
+/*
+ * Author: Dslyecxi, MikeMatrix, joko // Jonas
+ * Bind all required EventHandlers to Player map.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * _mapCtrl call ace_map_gestures_fnc_initDisplayDiary
+ *
+ * Public: No
+ */
+params ["_mapCtrl"];
+TRACE_1("initDisplayDiary",_mapCtrl);
+
+if (!isNil QGVAR(DrawMapHandlerID)) then {
+    _mapCtrl ctrlRemoveEventHandler ["Draw", GVAR(DrawMapHandlerID)];
```

Comment:
this breaks interaction menu on map (probably random what it breaks) because it's using gvar and runs multiple times
```
22:14:01 Added 1 // the briefing (like rscGetReady?)

22:14:09 Removing 1 // mission map (RscMainBreifing?) - the 1 was the ace_interact draw eh
22:14:09 Added 3
```